### PR TITLE
fix judgment on spiderpoolperformance.json file

### LINKED ISSUE #2530: Night CI 2023-11-02: Failed
action url: https://github.com/spidernet-io/spiderpool/actions/runs/6737772961

### PR DESCRIPTION
**Pull request by ty-dc**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #2530

**Special notes for your reviewer**:

### CODE REVIEW

**Review by Icarus9913 — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/doc.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: Documentation issue
 description: help us improve doc
-labels: kind/doc
+labels: [kind/doc]
```
Comment (cyclinder) [inline .github/ISSUE_TEMPLATE/doc.yaml:3]: labels: ["kind/doc"]

Comment (ty-dc) [reply .github/ISSUE_TEMPLATE/doc.yaml]: done

File: .github/workflows/e2e-init.yaml [hunk 2]
```diff
@@ -205,7 +205,7 @@ jobs:
           TOOL_PATH=${{ env.PERFORMANCESCRIPT }}
           TODO_STATUS=$( ${TOOL_PATH}  "e2ereport.json"  )
           echo "PERFORMANCE_RESULT=${TODO_STATUS}" >> $GITHUB_ENV
-          [ -f "spiderpoolperformance.json" ] || { echo "error, miss spiderpoolperformance.json " ; exit 1 ; }
+          [ -z ${{ env.PERFORMANCE_RESULT }} ] || { echo "error, miss spiderpool performance result " ; exit 1 ; }
```
Comment (weizhoublue) [inline .github/workflows/e2e-init.yaml:208]: 217行是 spiderpoolperformance.json ， 是否全部能对齐

Comment (ty-dc) [reply .github/workflows/e2e-init.yaml:208]: 参考 [action](https://github.com/Schneegans/dynamic-badges-action)， 217 的filename，并不是在找文件，而是帮助生成一个为 spiderpoolperformance.json 的文件。

The target filename - each gist may contain several files. This should have a .json extension if you want to use shields.io/endpoint or an .svg extension if you want to use the SVG mode described above.

Comment (weizhoublue) [reply .github/workflows/e2e-init.yaml:208]: 我弄错了，json 是  badge 的更新目标

**Pull request merged**